website/docs/r/security_tag.html.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: security_tag"
+sidebar_current: "docs-vcd-resource-security-tag"
+description: |-
+Provides a VMware Cloud Director Security Tag resource. This can be
+used to assign security tag to VMs.
+---
+
+# vcd\_security\_tag
+
+Provides a VMware Cloud Director Security Tag resource. This can be
+used to assign security tag to VMs.
+
+Supported in provider *v3.7+* and requires VCD 10.3.1+
+
+-> **Note:** This resource requires either system or org administrator privileges.
+
+## Example Usage
+
+```hcl
+resource "vcd_security_tag" "my_tag" {
+  name = "test-tag"
+  vm_ids = [vcd_vm.my-vm-one.id, vcd_vm.my-vm-two.id]
+}
+```
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the security tag.
+* `vm_ids` - (Required) List of VM IDs that the security tag is going to be applied to.
+
+# Importing
+
+~> **Note:** The current implementation of Terraform import can only import resources into the state.
+It does not generate configuration. [More information.](https://www.terraform.io/docs/import/)
+
+An existing security tag can be [imported][docs-import] into this resource. An example is below:
+
+```
+terraform import vcd_security_tag.my-tag my-security-tag-name
```

Comment:
The question here remains - how do I specify `org` as I can see in UI that creating a `tag` in one organization does not show it in other one.

Reply:
Changed so now an org need to be specified